docs/embed/white-labelling.md

```diff
@@ -0,0 +1,142 @@
+- favicon.ico
+- n8n-logo.svg
+- n8n-logo-collapsed.svg
+- n8n-logo-expanded.svg
+
+Replace these logo assets used in different Vue.js components like
+
+* [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue) - top/left logo in the main sidebar
+* [Logo.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/Logo.vue) - Reused in other components
+
+In the following example replace `n8n-logo-collapsed.svg` and `n8n-logo-expanded.svg` to update the main sidebar's logo assets.
+
+![Example Logo Main Sidebar](/_images/embed/white-label/logo-main-sidebar.png)
+
+If your logo assets require different sizing or placement you can customize SCSS styles at the bottom of [MainSidebar.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/MainSidebar.vue).
+
+```scss
+.logoItem {
+	display: flex;
+	justify-content: space-between;
+	height: $header-height;
+	line-height: $header-height;
+	margin: 0 !important;
+	border-radius: 0 !important;
+	border-bottom: var(--border-width-base) var(--border-style-base) var(--color-background-xlight);
+	cursor: default;
+
+	&:hover, &:global(.is-active):hover {
+		background-color: initial !important;
+	}
+
+	* { vertical-align: middle; }
+	.icon {
+		height: 18px;
+		position: relative;
+		left: 6px;
+	}
+
+}
+```
+
+## Text Localization
+
+To change all text occurrences like `n8n` or `n8n.io` to your brand identity you can customize n8n's english internationalization file:
+
+- [packages/editor-ui/src/plugins/i18n/locales/en.json](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/plugins/i18n/locales/en.json)
+
+n8n uses the [Vue I18n](https://kazupon.github.io/vue-i18n/) internationalization plugin for Vue.js to translate the majority of UI texts. To search and replace text occurrences inside `en.json` you can use [Linked locale messages](https://kazupon.github.io/vue-i18n/guide/messages.html#linked-locale-messages).
+
+In the following example add the `_brand.name` translation key to white label n8n's [AboutModal.vue](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/AboutModal.vue).
+
+```js
+{
+	"_brand.name": "My Brand",
+	//replace n8n with link to _brand.name
+	"about.aboutN8n": "About @:_brand.name",
+	"about.n8nVersion": "@:_brand.name Version",
+}
+```
+
+![Example About Modal Localization](/_images/embed/white-label/about-modal.png)
+
+### Window Title
+
+To change n8n`s window title to your brand name look at
+
+- [packages/editor-ui/index.html](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/index.html)
+- [packages/editor-ui/src/components/mixins/titleChange.ts](https://github.com/n8n-io/n8n/blob/master/packages/editor-ui/src/components/mixins/titleChange.ts)
+
+In the following example replace all occurrences of `n8n` and `n8n.io` with `My Brand`.
+
+![Example Window Title Localization](/_images/embed/white-label/window-title.png)
```

Comment:
Should we also tell them what to change to get this effect? I think perhaps there should be a code snippet here?

Reply:
I'll look into it.

Reply:
Added code sample.